config/tailwind.config.js

```diff
@@ -7,7 +7,8 @@ module.exports = {
     './public/*.html',
     './app/helpers/**/*.rb',
     './app/javascript/**/*.js',
-    './app/views/**/*.{erb,haml,html,slim}'
+    './app/views/**/*.{erb,haml,html,slim}',
+    './node_modules/flowbite/**/*.js',
```

Comment:
Oh shoot, I totally removed `node_modues` -- @FerPerales is there no way to get flowbite without this?

Reply:
@littleforest based on the research we did in #20 , this is the recommended way to do it. We were reluctant to introduce the dependency only for closing flash messages, but I think @fatimaburke is considering using more elements

Reply:
@FerPerales but if we are loading flowbite through importmaps, can we access it through there instead?

Reply:
@littleforest what we are doing through importmaps is adding turbo support, according to the [docs](https://flowbite.com/docs/getting-started/rails/#importmap). We can remove it and see if it works